### DISCUSSION TITLE
Read PnP Template from blob storage

### DISCUSSION THREAD

**Discussion by Dipali-arup** [pnp/pnpframework#377] [Q&A]
Hello there,

SharePointPnPCoreOnline has XMLAzureStorageTemplateProvider   to get PnP Template from blob storage , however I would like to know what is equivalent to get template from blob storage using PnP.Framework?

I see XMLAzureStorageTemplateProvider is not available in PnP.Framework.

many thanks, 
regards,

**Comment by jansenbe** [comment]
@Dipali-arup : this feature was dropped from PnP.Framework to limit the number of dependencies. In the community an alternative was developed: https://github.com/Qubix365/pnpframework-azurestorageconnector
